packages/data-broker/src/__tests__/DataBroker/DataBroker.test.ts

```diff
@@ -166,35 +172,71 @@ describe('DataBroker', () => {
 
     describe('passes mapping to service', () => {
       const dataBroker = new DataBroker();
-      const mapping = new DataServiceMapping([
-        { dataSource: DATA_ELEMENTS.DHIS_01, service_type: 'dhis', config: {} },
-      ]);
 
-      const testData: [MethodUnderTest, any[]][] = [
-        ['push', [{ code: 'DHIS_01', type: 'dataElement' }, [{ value: 2 }], TO_OPTIONS]],
-        ['delete', [{ code: 'DHIS_01', type: 'dataElement' }, { value: 2 }, TO_OPTIONS]],
-        ['pullMetadata', [{ code: 'DHIS_01', type: 'dataElement' }, TO_OPTIONS]],
-      ];
+      it('passes mapping to service: push', async () => {
```

Comment:
This section is just breaking apart the parameterised tests into separate test blocks per API

I think parameterised tests should only be used for simple, pure functions where we want to test multiple variations for the same API, e.g. maths, date handling etc. They are probably overused in `tupaia` (guilty as charged!)